Gemfile.lock

```diff
@@ -166,14 +166,14 @@ GEM
     selenium-webdriver (3.142.7)
       childprocess (>= 0.5, < 4.0)
       rubyzip (>= 1.2.2)
-    spring (2.1.0)
+    spring (2.1.1)
```

Comment:
### [spring](https://github.com/rails/spring)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.1.0 → 2.1.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **15** commits. See the full changes on [the compare page](https://github.com/rails/spring/compare/v2.1.0...v2.1.1).

These are the first 10 commits:
- (8a3297f) [Add support for gems.rb and gems.locked, closes #524.](https://github.com/rails/spring/commit/8a3297fb778e06c047279acdea44ffdf5661e93e)
- (21c7dc6) [Test with Rails 6.0.0 rc1](https://github.com/rails/spring/commit/21c7dc6d7dfcd141bb0a92a858e4f18fa2e8f7aa)
- (7e57f3b) [Exclude ruby 2.4.6 +Rails 6.0.0.rc1](https://github.com/rails/spring/commit/7e57f3b0dc299bea16387f104dfce365639d2c0e)
- (f65c0b8) [Merge pull request #594 from robertomiranda/patch-1](https://github.com/rails/spring/commit/f65c0b87151f55540e9ac7c5cf71e4e86f18d61d)
- (2fc1d35) [Avoid -I rubylibdir with default-gem bundler](https://github.com/rails/spring/commit/2fc1d3585b341a4207733db95c2f6c77a52a78ca)
- (caa09ef) [Merge pull request #595 from k0kubun/default-gem-bundler](https://github.com/rails/spring/commit/caa09ef84270ffaf546b7cf4179ffaebed5f7cd0)
- (baa3c16) [CI: Drop unused sudo: false Travis directive](https://github.com/rails/spring/commit/baa3c16747c4c5d7748752f57aad0687f4fb99ac)
- (22df910) [Merge pull request #597 from olleolleolle/patch-1](https://github.com/rails/spring/commit/22df9100c81c21777f792facc2d4c72cd4aa1fef)
- (6e5057a) [Added information about the spring-commands-parallel-tests g…](https://github.com/rails/spring/commit/6e5057a4a0f82514f20427c2c99ee11ada4673fd)
- (ee68785) [Merge pull request #607 from DocSpring/spring-commands-paral…](https://github.com/rails/spring/commit/ee687859008e947bc905b95121e306e2948d31c9)


</details>